configuration/yaml.md

```diff
@@ -0,0 +1,338 @@
+| `tags`            | The list of tags. This can contain both semantic tags and non-semantic tags.                                                                                                                                                                                                     |
+| `format`          | A short form for the `stateDescription` `pattern` metadata. This is the equivalent of the format enclosed in square brackets in DSL's item label.                                                                                                                                |
+| `unit`            | A short form for the `unit` metadata, to define the item's unit.                                                                                                                                                                                                                 |
+| `autoupdate`      | A short form for the `autoupdate` metadata. Valid values are: `true` or `false`.                                                                                                                                                                                                 |
+| `channel`         | A short form for the `channels` list to define a linked channel UID for when there is only one channel with a blank config.                                                                                                                                                      |
+| `channels`        | A key-value map to define the channels linked from this item. The key is the channel uid to link to, and the value is the link configuration as a key-value map. When no configuration is required, an empty map `{}` must be used as the value.                                 |
+| `metadata`        | A key-value map to define item metadata. The metadata defined in here overrides any short forms when both are specified.                                                                                                                                                         |
+
+Example:
+
+```yaml
+version: 1
+
+items:
+  BedRoom1_Switch:
+    type: Group
+    label: Bedroom1 Light Bulb
+    groups:
+      - lBedroom1
+    tags:
+      - Lightbulb
+
+  BedRoom1_Light_Switch:
+    type: Switch
+    label: Bed Room1 Light Switch
+    icon: switch
+    format: '%s'
+    groups:
+      - BedRoom1_Light_Bulb
+    tags:
+      - Switch
+    channel: mqtt:topic:bedroom1-switch:switch1
+
+  BedRoom1_Switch_StatusLED:
+    type: Switch
+    groups:
+      - BedRoom1_Switch_Group
+    channels:
+      mqtt:topic:bedroom1-switch:statusled:
+        # The configuration of this item-channel link is defined below
```

Comment:
It could have been interesting to provide a more common example than profile, the case when the channel has a configuration parameter.

Reply:
Any channel that defines a configuration parameter.
I have in mind as an example one of both channels of the NTP binding where you can adjust through a parameter how will be formatted the day/hour.

Reply:
> https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.ntp/src/main/resources/OH-INF/thing/thing-types.xml#L68

Isn't that for the `Thing` channel configuration, whereas this is the item-channel-link which is part of the Item definition?

Reply:
Oups, I think you are right, sorry!

Reply:
AFAIK, item-channel-link config is currently only used by profiles, but it should be possible to use it for other things, such as the tagging we're discussing in the other issue, but for now only profiles use it.

Reply:
I'm leaving the config as is for now, but feel free to comment again if you have further suggestions